charts/zeebe-benchmark/test/golden/leader-balancing-cron.golden.yaml

```diff
@@ -0,0 +1,26 @@
+---
+# Source: zeebe-benchmark/templates/leader-balancing-cron.yaml
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: leader-balancer
+  labels:
+    app.kubernetes.io/name: zeebe-benchmark
+    app.kubernetes.io/instance: benchmark-test
+    app.kubernetes.io/version: "8.1.2"
+    app.kubernetes.io/managed-by: Helm
```

Comment:
:question: I'm wondering a bit where these labels are coming from. Is `zeebe-benchmark.labels` a special variable and the labels are coming from helm? 

In particular I'm not sure why version is `8.1.2`. Depending on the source we should maybe ignore it in the goldenfile comparison?

Reply:
Some are from helm yes, like the managed-by.

I think it make sense to have them here as you see you are wondering why this is part of it and the same applies if they change you would wonder which is the purpose. We want to see when something unexpected changes.

Speaking of this, I would say it should be part of it since this might also break things and should be done with care. If you change the version update the golden file and you see the reflection of it. But sure we can discuss this further regarding the version I mean, for the others I think we should definitively have them.